whichone/static/js/ui.js

```diff
@@ -1,170 +1,172 @@
 function cdwn() {
-  countdown -= 1;
-  if (countdown <= 0 && !paused) {
-    wrongAnswer(true);
-  }
-  countdownNumberEl.textContent = countdown;
+    countdown -= 1;
+    if (countdown > 0 && countdown <= 6 && !paused && !stopped) {
+        $("#inner-circle").css("stroke", "rgb(239,83,80)");
+    } else {
+        $("#inner-circle").css("stroke", "rgb(255,199,137)");
+    }
+    if (countdown == 0 && !paused) {
+        wrongAnswer(null);
+    }
+    countdownNumberEl.textContent = countdown;
 }
 
 function startCounter() {
-  $("#countdown-number").css("display", "block");
-  $("#inner-circle").css("display", "block");
-  countdown = 10;
-  countdownNumberEl.textContent = countdown;
-  if (refreshIntervalId != null) {
-    clearInterval(refreshIntervalId);
-  }
-  refreshIntervalId = setInterval(cdwn, 1000);
-  $("#inner-circle").addClass("animated");
+    $("#countdown-number").css("display", "block");
+    $("#inner-circle").css("display", "block");
+    countdown = 12;
```

Comment:
This should probably be a injectable config so you only need to change it in 1 place in future if you change the countdown timer again.